problems/3.conditionals.js

```diff
@@ -2,25 +2,36 @@
 * @param {number} temp - The current temperature
 * @returns {string} - A message about the temperature
 */
-function temperatureMessage() {
+function temperatureMessage(temp) {
+  if (temp = -32) {
+    console.log("It's freezing")
+  } else {
+    console.log("It's alright")
+
+  }
```

Comment:
Want to return instead of console.log. Return makes the function evaluate to something. Console.log is just for a human to see something and evaluates to undefined.